### PR TITLE
文档调整

### PR DESCRIPTION
**Pull request by CoderOverflow**
文档调整

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +33,18 @@ Download the jars from the release page: [Release页面](https://github.com/allw
 Step 2:
 
 Visit the downloads page: [Spark](https://spark.apache.org/downloads.html), to download Apache Spark 2.2.0 and then unarvhive it.
- 
+
+2.1 unarvhive the Apache Spark 2.2.0 package
```
Comment (allwefantasy) [inline README.md]: We can show clearly that these steps are optional cause we can use  full path e.g.  ./[your-spark-home]/bin/submit ...  to submit our application.

**Review by allwefantasy — COMMENTED** [review]
Can you merge all this commits to one commit then invoke PR?

File: README.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ tar -zxvf spark-2.2.0-bin-hadoop2.7.tgz
 ```shell
 vi /etc/profile
 ```
-  add following sentences at the end of the file
+    add following sentences at the end of the file
```
Comment (allwefantasy) [inline README.md]: o

**Review by allwefantasy — CHANGES_REQUESTED** [review]

**Comment by CoderOverflow** [comment]
OK， let me try  ：）

**Pull request closed without merging**